tests/test_json.py

```diff
@@ -0,0 +1,72 @@
+import os
+import pathlib
+import tempfile
+import json
+
+from substrait.proto import Plan
+from substrait.json import load_json, parse_json, dump_json, write_json
+
+import pytest
+
+
+JSON_FIXTURES = (
+    pathlib.Path(os.path.dirname(__file__))
+    / ".."
```

Comment:
I preferred to go for `/ ".."` because I find it more readable than invoking methods when expressing a path, but happy to go for `parent` if there is a preference

Reply:
Fine with me.